paasta_tools/cli/cmds/local_run.py

```diff
@@ -381,6 +381,37 @@ def get_container_name():
     return 'paasta_local_run_%s_%s' % (get_username(), randint(1, 999999))
 
 
+def check_for_sh(command):
```

Comment:
This feels rather hackish and error-prone (i.e. what if they use a different shell).